src/styles/block/cards.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  background: #fff;
```

don't forget to create variables for colors
Create for all colors
```suggestion
  background: $c-white;
```